### PR TITLE
Promote qulacs to an endorsed plugin, and add in default.qubit.autograd

### PR DESCRIPTION
**Pull request by josh146**
Note: I combined `default.qubit.tf` and `default.qubit.autograd` into a single card, as 5 cards were too many to fit in a single row. It doesn't look great, and I'm unsure how to include multiple links on a card. Open to any ideas!

### CODE REVIEW

**Comment by mariaschuld** [comment]
I would definitely make two cards and start a new row - that just looks like more. Nothing wrong with to 3+2 cards a row IMO, then it looks like things are still growing :)

Great to have one more endorsed device!

File: jinja/plugins.html [hunk 1]
```diff
@@ -111,15 +114,12 @@ <h4 class="card-header weird-blue-gradient">
...
-                    <img class="img-fluid plugin-logo" src="img/qsharp.png" style="max-height: 25px">
-                </h4>
+            <a class="card plugin-card" href="https://pennylane-qulacs.readthedocs.io">
+                <h4 class="card-header weird-blue-gradient">Qulacs</h4>
                 <div class="card-body">
                     <p class="card-text">
-                        Microsoft QDK is a library for quantum programming using the .NET Q# quantum programming language.
-                        Provides access to the QDK full state simulator to be used with PennyLane.
+                        Qulacs is a high-performance quantum circuit simulator for simulating large, noisy or parametric quantum circuits.
+                        Implemented in C/C++ and with python interface, Qulacs achieved both high speed circuit simulation and high usability.
```
Comment (trbromley) [inline jinja/plugins.html:122]:
```suggestion
                        Implemented in C/C++ and with Python interface, Qulacs achieved both high speed circuit simulation and high usability.
```

File: jinja/plugins.html [hunk 2]
```diff
@@ -25,17 +25,19 @@ <h4 class="card-header juicy-peach-gradient">default.qubit</h4>
...
+                            <br><br>A good choice for optimizations with a moderate number of qubits and parameters, and stochastic
+                            expectation values.
                         </p>
                     </div>
                 </a>
             </div>
             <div class="card plugin-card">
                 <a href="https://pennylane.readthedocs.io/en/stable/code/api/pennylane.devices.default_qubit_tf.DefaultQubitTF.html">
-                    <h4 class="card-header juicy-peach-gradient">default.qubit.tf</h4>
+                    <h4 class="card-header juicy-peach-gradient">default.qubit.tf<br> default.qubit.autograd</h4>
```
Comment (trbromley) [inline jinja/plugins.html:36]: Also support having an extra card, it looks a bit too busy in the title area:
image

Although yeah the layout may be tough :thinking:

File: jinja/plugins.html [hunk 3]
```diff
@@ -25,17 +25,19 @@ <h4 class="card-header juicy-peach-gradient">default.qubit</h4>
                     <div class="card-body">
                         <p class="card-text">
                             A simple state-vector qubit simulator written in Python.
-                            <br><br>A good choice for optimizations with a moderate number of qubits and parameters.
+                            <br><br>A good choice for optimizations with a moderate number of qubits and parameters, and stochastic
+                            expectation values.
```
Comment (trbromley) [inline jinja/plugins.html:29]: How about something like "or when using stochastic expectation values"? I'm worries the "and" might make it seem like you have to be in non-analytic mode.

Comment (josh146) [reply jinja/plugins.html:29]:
```suggestion
                            <br><br>A good choice for optimizations with a moderate number of qubits and parameters, or when using stochastic
                            expectation values.
```

File: jinja/plugins.html [hunk 4]
```diff
@@ -46,7 +48,8 @@ <h4 class="card-header juicy-peach-gradient">lightning.qubit (beta)</h4>
                     <div class="card-body">
                         <p class="card-text">
                             A fast state-vector qubit simulator written with a C++ backend.
-                            <br><br>A good choice for optimizations with a moderate number of qubits and parameters.
+                            <br><br>A good choice for optimizations with a moderate number of qubits and parameters, and stochastic
+                            expectation values.
```
Comment (josh146) [inline jinja/plugins.html:52]:
```suggestion
                            <br><br>A good choice for optimizations with a moderate number of qubits and parameters, or when using stochastic
                            expectation values.
```

File: jinja/plugins.html (hunk 2 above)
Comment (josh146) [reply jinja/plugins.html:36]: Yeah, I'm too much of a perfectionist, and couldn't work out a layout that looked nice! adding an extra row also looked quite ugly to me, and added additional vertical scrolling to the page.

**Comment by josh146** [comment]
Two rows 🤔 

image

**Comment by co9olguy** [comment]
Likely a bit more work, but I keep thinking about Monopoly, and how if you have 2-3 properties of the same colour/type, you stack them on top of eachother with only the headers showing. 

http://monopolydealrules.com/photos/Monopoly-Deal-Winning-Hand-with-wildcards.png


We could do something similar here for plugin "families", but when you click on one of the lower cards, it's full text is shown

**Review by mariaschuld — APPROVED** [review]
Thanks Josh! 2 lines looks much better, could even keep all cards the same width as Tom said. But not so crucial I think!

**Comment by josh146** [comment]
> Thanks @josh146, looks good to me! Minor question - is it hard to get the width of the tiles in the second row to match the width of the tiles in the first?

Nope, just requires more work - the card deck needs to be replaced with rows and columns, and the spacing done manually

> Likely a bit more work, but I keep thinking about Monopoly, and how if you have 2-3 properties of the same colour/type, you stack them on top of eachother with only the headers showing. 
> 
> 
> Monopoly-Deal-Winning-Hand-with-wildcards
> 
> We could do something similar here for plugin "families", but when you click on one of the lower cards, it's full text is shown

This is really awesome lol

**Pull request merged**